gpii/node_modules/processReporter/src/ProcessReporter.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * GPII Process Reporter
+ *
+ * Copyright 2015 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("kettle", require);
+fluid.require("./processesBridge.js", require);
+
+fluid.defaults("gpii.processReporter", {
+    gradeNames: "fluid.component"
+});
+
+fluid.defaults("gpii.processReporter.base", {
+    gradeNames: ["kettle.app", "gpii.processReporter"],
+    requestHandlers: {
+        processGet: {
+            route: "/processes",
+            type: "gpii.processReporter.allSolutions.get",
+            method: "get"
+        },
+        processGetSingle: {
+            route: "/processes/:solutionID",
+            type: "gpii.processReporter.solution.get",
+            method: "get"
+        }
+    },
```

Comment:
Hey @kaspermarkus Thanks for the update.  My guess is your use of the PR is in your GPII-1230 branch?  I'll take a look.